dbcon/mysql/ha_mcs_pushdown.cpp

```diff
@@ -21,23 +21,12 @@
 
 void check_walk(const Item* item, void* arg);
 
-void disable_indices_for_CEJ(THD *thd_)
+void restoreQueryState(ha_columnstore_select_handler* handler)
```

Comment:
I would prefer to stick with the consistent naming schema in this unit.

Reply:
I agree. Updated the name.